package.json

```diff
@@ -24,6 +24,7 @@
     "babel-core": "~5.5.6",
     "babel-jest": "~5.3.0",
     "babel-loader": "~5.1.4",
+    "file-loader": "git+https://github.com/webpack/file-loader.git",
```

Comment:
curious why this pulls from a URL instead of a version number like the rest?

The others got added by "npm install FOO -save"